### PR TITLE
Add basic test

### PR DESCRIPTION
**Pull request by KhoaKoha**
Using webDriverWait explicit wait 
Only one test case

### CODE REVIEW

File: src/main/java/com/utils/BasicTest.java [hunk 1]
```diff
@@ -15,20 +15,14 @@ public abstract class BasicTest {
...
-        // ChromeOptions options = new ChromeOptions();
-        // System.setProperty("webdriver.chrome.driver", driverPath);
-        // driver = new ChromeDriver(options);
+
         WebDriverManager.chromedriver().setup();
         driver = new ChromeDriver();
-        // Maximize the browser
         driver.manage().window().maximize();
-        //driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
```
Comment (duykhanhrc) [inline src/main/java/com/utils/BasicTest.java:25]: Nếu mình xài explicitly wait thì không  xai cái này. Remove it

File: src/test/java/com/LoginTest.java [hunk 2]
```diff
@@ -1,19 +1,54 @@
...
     public void loginTest() throws Exception {
-        // Launch website
-        String url = "https://seniorautomationtest.com/";
+        WebDriverWait wait = new WebDriverWait(driver, 10);
+        String url = "https://bantheme.xyz/hathanhauto/tai-khoan/";
         driver.get(url);
         Assert.assertEquals(driver.getCurrentUrl(), url);
-    }
 
+        WebElement submitButton = driver.findElement(By.xpath("//button[@name='register']"));
```
Comment (duykhanhrc) [inline src/test/java/com/LoginTest.java:25]: Looks good. Ma refactor xai POM minh coi phat

**Comment by duykhanhrc** [comment]
Hi,

Your homework 16 has been reviewed, thank you for your effort.
Great! Keep up the good work!

Best wishes from Automation Test Pro,
Khanh

5A26A6A1-9479-4A5A-9F74-B99BBDCB97A9

File: src/main/java/com/utils/BasicTest.java (hunk 1 above)
Comment (KhoaKoha) [reply src/main/java/com/utils/BasicTest.java:25]: Đã remove

**Pull request closed without merging**